### ISSUE TITLE
Sybil Attacker Report

### ISSUE THREAD

**Issue by GreyPK** [hop-protocol/hop-airdrop#207]
# Related Addresses

15 related addresses. 
```
0x5691df0ddc35ce430856a333566e2b19b7c9abcc
0x80d19ec3a87f45e83c33547d8fb56f51266419b9
0xb713df754bbf37db676e2b49d62d5f9532b56746
0xb361e41e92870c8a05587665c73aa7b8c5f5f609
0xbd985b4632a3ad19ad54cb75db7c3fb53a490fb8
0xcb78dac9a2c43c62e454d62184f72373917e3ae7
0xc83a13319561e3c080c9b8ef31b8ecf6c1e63c68
0x16326d7b00cb6175fe97631ac6e957ce346d1643
0x1cccff3de9b0a7c466acb31f33910691f1cb6e14
0xa32a1f1c227b7b5d8251465f9398ac70675af6d8
0xaea79e61d658690b2cc0c4078d37ab18cfd2afaf
0x4f647966968b1f417f2181327d333575ad0b46d3
0xcc06ac76b8e778d75af1cecc57561c0197160f68
0x8aa3e3da214d410440ec56814b5999b4f1d15c82
0x8752c42677daa72fd1c54f8bb480f5cf0b8bfda3
```

# Reasoning

1) An elligible for the airdrop Address 0x6c13fc8ef50144a21702415d57ece0a09879fcfb send MATIC to these 15 addresses that elligible for the airdrop. These transactions can find here: https://polygonscan.com/txs?a=0x6c13fc8ef50144a21702415d57ece0a09879fcfb&f=2&ps=100&p=7
2) All 15 wallets first used HOP with SwapAndSend in almost same date (15-25 January 2022).
3) All 15 wallets used HOP swapAndSend many times, mostly 10 times and total swap volume ~$1400-2500$, mostly $2200.

0x5691df0ddc35ce430856a333566e2b19b7c9abcc used HOP swapAndSend January-15-2022 2 times: 1194 USDT and 1348 USDC, total $2542
https://blockscout.com/xdai/mainnet/address/0x5691DF0DdC35ce430856A333566E2b19B7C9aBCC/transactions (search 0xeea0d7b2)

0x80d19ec3a87f45e83c33547d8fb56f51266419b9 used HOP swapAndSend January-17-2022 8 times total 2124 xDAI
https://blockscout.com/xdai/mainnet/address/0x80d19Ec3A87F45e83c33547D8Fb56F51266419b9/transactions?filter=from

0xb713df754bbf37db676e2b49d62d5f9532b56746 used HOP swapAndSend January-15-2022 6 times total 1378 USDT
https://blockscout.com/xdai/mainnet/address/0xb713Df754bbF37db676e2B49d62d5f9532B56746/transactions?filter=from

0xb361e41e92870c8a05587665c73aa7b8c5f5f609 used HOP swapAndSend January-19-2022 10 times total 2305 xDAI
https://blockscout.com/xdai/mainnet/address/0xb361e41e92870C8a05587665C73AA7B8c5f5f609/transactions?filter=from

0xbd985b4632a3ad19ad54cb75db7c3fb53a490fb8 used HOP swapAndSend January-19-2022 10 times total 2271 xDAI
https://blockscout.com/xdai/mainnet/address/0xbd985b4632a3AD19AD54CB75DB7c3FB53a490FB8/transactions?filter=from

0xcb78dac9a2c43c62e454d62184f72373917e3ae7 used HOP swapAndSend January-20-2022 10 times total 2160 xDAI
https://blockscout.com/xdai/mainnet/address/0xCb78daC9A2C43C62e454D62184F72373917E3ae7/transactions?filter=from

0xc83a13319561e3c080c9b8ef31b8ecf6c1e63c68 used HOP swapAndSend January-20-2022 10 times total 2193 xDAI
https://blockscout.com/xdai/mainnet/address/0xC83A13319561e3c080C9b8eF31B8ecF6C1E63c68/transactions?filter=from

0x16326d7b00cb6175fe97631ac6e957ce346d1643 used HOP swapAndSend January-23-2022 10 times total 2113 xDAI
https://blockscout.com/xdai/mainnet/address/0x16326D7b00cB6175Fe97631Ac6e957CE346D1643/transactions?filter=from

0x1cccff3de9b0a7c466acb31f33910691f1cb6e14 used HOP swapAndSend January-24-2022 10 times total about 2100 xDAI
https://blockscout.com/xdai/mainnet/address/0x1ccCfF3DE9B0A7c466acB31F33910691f1CB6E14/transactions?filter=from

0xa32a1f1c227b7b5d8251465f9398ac70675af6d8 used HOP swapAndSend January-25-2022 13 times total about 2500 USDC 
https://blockscout.com/xdai/mainnet/address/0xa32A1F1c227b7B5d8251465f9398AC70675af6D8/transactions?filter=from

0xaea79e61d658690b2cc0c4078d37ab18cfd2afaf used HOP swapAndSend January-25-2022 12 times total about 2500 USDC
https://blockscout.com/xdai/mainnet/address/0xaea79E61d658690B2cc0C4078D37AB18cFD2aFAF/transactions?filter=from

0x4f647966968b1f417f2181327d333575ad0b46d3 used HOP swapAndSend January-26-2022 12 times total about 2500 USDC
https://blockscout.com/xdai/mainnet/address/0x4F647966968b1F417F2181327d333575aD0b46d3/transactions?filter=from

0xcc06ac76b8e778d75af1cecc57561c0197160f68 used HOP swapAndSend January-21-2022 10 times total about 2200 xDAI
https://blockscout.com/xdai/mainnet/address/0xCc06aC76b8e778d75AF1CECc57561c0197160F68/transactions?filter=from

0x8aa3e3da214d410440ec56814b5999b4f1d15c82 used HOP swapAndSend January-21-2022 10 times total about 2100 xDAI
https://blockscout.com/xdai/mainnet/address/0x8AA3e3Da214d410440Ec56814b5999b4f1D15C82/transactions?filter=from

0x8752c42677daa72fd1c54f8bb480f5cf0b8bfda3 used HOP swapAndSend January-19-2022 10 times total about 2200 xDAI
https://blockscout.com/xdai/mainnet/address/0x8752c42677DAA72fD1C54F8BB480F5cf0b8BfdA3/transactions?filter=from

# Methodology

I checked polygon's matic transactions with polygonscan API and filtered transactions, where 15+ recipients addresses and sende are eligible for the airdrop. Then I checked when and how this wallets interact with HOP. Node js code available on request.

# Rewards Address

0x7d3f6048091aEdE6198736b8C6ac527c314e25ff

**Comment by dai114** [comment]
I am a community manager. About 100 members of the community have increased in recent years; I have been leading you to learn and experience this project; These 15 addresses belong to different members (perhaps a person may have 2-5 addresses);

Our working mode is that when I find some projects worthy of interaction, I will publish relevant information and simple tutorials in the community. At this point, some loyal members will interact immediately, which is why these addresses will be concentrated for a period of time;

Since most of my community members are novices in blockchain, I will distribute some matic and even loan funds to them;

Therefore, it is worthless in terms of issuance records;

I don't like air drop hunters either. They occupy too many airdrop quotas. At the same time, they will throw all tokens immediately after the air drop, which has a great impact on the project;

I hope the team can understand my working mode. Our interaction behavior is essentially different from that of air drop hunters; As far as I know, the behavior of air drop hunters is automatic (the same number and amount), large-scale (100 and above addresses) and centralized (concentrated in one day); Our behavior is only a small concentration (usually within 1-15 days)

My goal is to attract more members and establish a Dao organization to learn and grow together; Please don't misunderstand my members, thank you;

**Comment by pai1232** [comment]
0xb361e41e92870c8a05587665c73aa7b8c5f5f609   this is my address，Is there anything wrong with that?

**Comment by pai1232** [comment]
There are several addresses above that I am very familiar with. We know each other in reality;
Our English is not good, so we seldom use discard and GitHub; But that doesn't mean you can report at will and no one refutes.

**Comment by chengfengwuyue** [comment]
0x80d19ec3a87f45e83c33547d8fb56f51266419b9
this is my address，Is there anything wrong with that?

**Comment by DW5566** [comment]
0xcb78dac9a2c43c62e454d62184f72373917e3ae7
this is my address，are you OK？

**Comment by 77897-afk** [comment]
0x4f647966968b1f417f2181327d333575ad0b46d3  my address
We don't accept slander

**Comment by boasdt1** [comment]
0xc83a13319561e3c080c9b8ef31b8ecf6c1e63c68
my address im not a sybil
what are you doing？？

**Comment by hellobull** [comment]
0x5691df0ddc35ce430856a333566e2b19b7c9abcc
this is my address；

**Comment by GreyPK** [comment]
All wallets used HOP to bridge only between Polygon and Gnosis 15-26 January 2022, only stablecoins, which he forwarded in turn to his next addresses! The most common transaction volume is ~$200. 

0x5691df0ddc35ce430856a333566e2b19b7c9abcc 15 January 2022 times: total USDT ~$2370
After he send USDT to 0xb713Df754bbF37db676e2B49d62d5f9532B56746
https://blockscout.com/poa/xdai/tx/0x4908fcfb15c97b1c32dd8add84cf79da8d64550db54f562c6d566a56d02890b1

0xb713df754bbf37db676e2b49d62d5f9532b56746 15-16 January 2022 7 times: total USDT ~$2350
After he swapped USDT to USDC and send it to 0x80d19Ec3A87F45e83c33547D8Fb56F51266419b9
https://polygonscan.com/tx/0xe2bae1c8beb7a28f7737309b1ec557e3d86f1f04ae443b6ff598f5009277f6b3
https://polygonscan.com/tx/0xdeb0f1cd89571d359462d996a75950aeeae0d7e5fdfa3d212edc0887acb48e11

0x80d19ec3a87f45e83c33547d8fb56f51266419b9 17 January 2022 8 times: total DAI ~$2290
After he send DAI to sender address 0x6c13fc8ef50144a21702415d57ece0a09879fcfb, which send it to 0x1100cc5c25BEfe926D03d3375D3e813D34e76a8E, which send it to 0xb361e41e92870C8a05587665C73AA7B8c5f5f609 https://polygonscan.com/tx/0xdeb0f1cd89571d359462d996a75950aeeae0d7e5fdfa3d212edc0887acb48e11
https://polygonscan.com/tx/0xf510318187cf703c044638a83d28e16d0ed06cba4fe535ea7aa74405a99568c5
https://polygonscan.com/tx/0x711715607e027140e5a75910d10faeb766df89da580fa2e6c9483d249a77ee7e

0xb361e41e92870c8a05587665c73aa7b8c5f5f609 19 January 2022 10 times: total USDT ~$2290
After send DAI to 0xbd985b4632a3AD19AD54CB75DB7c3FB53a490FB8
https://polygonscan.com/tx/0xc8c3b74c2d9e8386f2ea104616a4b910116af4f9a2e5a7fbd8965de58b47ed1c

0xbd985b4632a3ad19ad54cb75db7c3fb53a490fb8 19 January 2022 10 times: total DAI ~$2260
After send DAI to 0x8752c42677DAA72fD1C54F8BB480F5cf0b8BfdA3
https://polygonscan.com/tx/0x35ee19def9f6168ce275d5d5d1c6c70296a033a1ceabe9221f7655806d373aaa

0x8752c42677daa72fd1c54f8bb480f5cf0b8bfda3 19-20 January 2022 10 times: total DAI ~$2215
After send DAI to 0xC83A13319561e3c080C9b8eF31B8ecF6C1E63c68
https://polygonscan.com/tx/0x4f82e3550057cd4a433774f11d937d46638c83588c0bf66153492c17e589bbb3

0xc83a13319561e3c080c9b8ef31b8ecf6c1e63c68 20 January 2022 10 times: total DAI ~$2180
After send DAI to 0xCb78daC9A2C43C62e454D62184F72373917E3ae7
https://polygonscan.com/tx/0xd56f138c1a4b529ec3f4ef40522a1ea3f66c1423440b1844569494a9cb41b89c

0xcb78dac9a2c43c62e454d62184f72373917e3ae7 20-21 January 2022 10 times: total DAI ~$2150
After send DAI to 0xCc06aC76b8e778d75AF1CECc57561c0197160F68
https://polygonscan.com/tx/0x5aa913de583a59dbcd0880a5d5bc709862fc697b6592d38b5fe12acb3c4383e0

0xcc06ac76b8e778d75af1cecc57561c0197160f68 21 January 2022 10 times: total DAI ~$2115
After send DAI to 0x8aa3e3da214d410440ec56814b5999b4f1d15c82
https://polygonscan.com/tx/0x55270f433e43cbe677ee4a552c4e0c33641f7795c7c2ecd8ebf2458213d42625

0x8aa3e3da214d410440ec56814b5999b4f1d15c82 21 January 2022 10 times: total DAI ~$2070
After send DAI to 0x16326D7b00cB6175Fe97631Ac6e957CE346D1643
https://polygonscan.com/tx/0x727bafc0f57368aa0b3176c89183806200edd45b56700999ed3fc815f8cb15af

0x16326d7b00cb6175fe97631ac6e957ce346d1643 23 January 2022 10 times: total DAI ~$2100
After send DAI to 0x1cccff3de9b0a7c466acb31f33910691f1cb6e14
https://polygonscan.com/tx/0x4c72d7845adf1f248d9bc6a62d3d6b8240e7d54eb22420a0370be370ff7f88ad

0x1cccff3de9b0a7c466acb31f33910691f1cb6e14 24-25 January 2022 10 times: total DAI ~$2110
After send DAI to 0xa32A1F1c227b7B5d8251465f9398AC70675af6D8, which swapped DAI to USDC
https://polygonscan.com/tx/0xb4261bd71c831caaf0b8e8a67eb26a41da75114750a06230dbe04653e9d632f6
https://polygonscan.com/tx/0xcaa4ecae2b1a99561d1ceb07738ea3426667243465b85c83c1bca943dfa26904

0xa32a1f1c227b7b5d8251465f9398ac70675af6d8 25 January 2022 13 times: total USDC ~$2660
After send USDC to 0xaea79E61d658690B2cc0C4078D37AB18cFD2aFAF
https://polygonscan.com/tx/0x11a7d5de02e81e97f503f765854c2e3c7ea56dbb939e89c3137527be9af7e46e

0xaea79e61d658690b2cc0c4078d37ab18cfd2afaf 25 January 2022 12 times: total USDC ~$2645
After send USDC to 0x4f647966968b1f417f2181327d333575ad0b46d3
https://polygonscan.com/tx/0x0bd17c004639e85394f235b80d51ca981de10aed0c072965cd012891b52284d3

0x4f647966968b1f417f2181327d333575ad0b46d3 26 January 2022 12 times: total USDC ~$2585
After send USDC to 0xc83a13319561e3c080c9b8ef31b8ecf6c1e63c68
https://polygonscan.com/tx/0xa89ca8235fd139290c1c8ecceb9c21c14384d59bb953a09dcec3bf6af0f51d01


https://explorer.hop.exchange/?account=0x5691df0ddc35ce430856a333566e2b19b7c9abcc
https://explorer.hop.exchange/?account=0x80d19ec3a87f45e83c33547d8fb56f51266419b9
https://explorer.hop.exchange/?account=0xb713df754bbf37db676e2b49d62d5f9532b56746
https://explorer.hop.exchange/?account=0xb361e41e92870c8a05587665c73aa7b8c5f5f609
https://explorer.hop.exchange/?account=0xbd985b4632a3ad19ad54cb75db7c3fb53a490fb8
https://explorer.hop.exchange/?account=0xcb78dac9a2c43c62e454d62184f72373917e3ae7
https://explorer.hop.exchange/?account=0xc83a13319561e3c080c9b8ef31b8ecf6c1e63c68
https://explorer.hop.exchange/?account=0x16326d7b00cb6175fe97631ac6e957ce346d1643
https://explorer.hop.exchange/?account=0x1cccff3de9b0a7c466acb31f33910691f1cb6e14
https://explorer.hop.exchange/?account=0xa32a1f1c227b7b5d8251465f9398ac70675af6d8
https://explorer.hop.exchange/?account=0xaea79e61d658690b2cc0c4078d37ab18cfd2afaf
https://explorer.hop.exchange/?account=0x4f647966968b1f417f2181327d333575ad0b46d3
https://explorer.hop.exchange/?account=0xcc06ac76b8e778d75af1cecc57561c0197160f68
https://explorer.hop.exchange/?account=0x8aa3e3da214d410440ec56814b5999b4f1d15c82
https://explorer.hop.exchange/?account=0x8752c42677daa72fd1c54f8bb480f5cf0b8bfda3

**Comment by dai114** [comment]
Yes, because I told them what to do. They need to use hop more than 10 times, but I don't know how many people will do it

**Comment by GreyPK** [comment]
[dai114](https://github.com/dai114) The 15 address transfer $1k in a circle, used it in a HOP bridge, and send it on. Of course everything is fair, it's just that you have a common family budget!
Or no, it's just newbies in crypto of your community that you give $1k to and tell them to pass it on to others after used HOP. Funny :)

**Comment by dai114** [comment]
Yes, we use "Solitaire" in other communication apps, because we have known these people for a long time, so we don't worry about the safety of property; This is why there are only 15 addresses;

For those who have just joined, I will only tell them "you need to use hop";

**Comment by Moooncheri** [comment]
0x8752c42677daa72fd1c54f8bb480f5cf0b8bfda3 This is my address

**Comment by GreyPK** [comment]
Solved in #377
